docs_zh_CN/manual/cluster-admin/installation-guide.md

```diff
@@ -103,7 +103,7 @@ openpai-004,10.0.0.4
 user: <your-ssh-username>
 password: <your-ssh-password>
 branch_name: pai-1.4.y
-docker_image_tag: v1.4.0
```

Comment:
Do we also need to change EN doc?

Reply:
> Do we also need to change EN doc?

Docker_image_tag is currently v1.5.0 because v1.4.x doesn't fit the master branch code.